transaction/executer_test.go

```diff
@@ -3,7 +3,6 @@ package transaction
 import (
```

Comment:
All removed lines are replaced with this: https://github.com/giantswarm/microkit/blob/983e16add51e2027056de0cc62079dc148de35f2/transaction/executer_test.go#L235-L321

Diff is ugly.